No luck with Uborder on macOS ARM64 nor Android ARM64

macOS:

```
2026-08-24 22:12:22.735 T:22789798   error <general>: CShaderGL::Create: Vertex shader compile error:
                                                   ERROR: 0:1: '' :  version '120' is not supported
                                                   ERROR: 0:2: '' :  #version required and missing.
                                                    
2026-08-24 22:12:22.736 T:22789798   error <general>: CShaderGL::Create: Fragment shader compile error:
                                                   ERROR: 0:1: '' :  version '120' is not supported
                                                   ERROR: 0:2: '' :  #version required and missing.
                                                   ERROR: 0:271: '' :   Macro Redefined COMPAT_VARYING
                                                    
2026-08-24 22:12:22.736 T:22789798   error <general>: CShaderGL::Create: Shader program link error:
                                                   ERROR: One or more attached shaders not successfully compiled
                                                    
2026-08-24 22:12:22.736 T:22789798   error <general>: CShaderGL::Create: Failed to load video shader: /Applications/Kodi.app/Contents/Resources/Kodi/addons/game.shader.presets/resources/glsl/bezel/uborder/base_presets/uborder-bezel-reflections/../../shaders/content_bezel_shaders/crt-nobody-bezel-reflections.glsl
2026-08-24 22:12:22.736 T:22789798   error <general>: CShaderPresetGL::CreateShaders: Couldn't create a video shader
2026-08-24 22:12:22.736 T:22789798 warning <general>: CShaderPreset::SetShaderPreset: Failed to initialize shaders
```

Android:

```
08-24 22:08:48.886 30370 30592 E Kodi    : 2026-08-24 22:08:48.886 T:30592   error <general>: CShaderGLES::Create: Vertex shader compile error:
08-24 22:08:48.886 30370 30592 E Kodi    :                                                    0(185) : error C1059: non constant expression in initialization
08-24 22:08:48.886 30370 30592 E Kodi    :                                                    0(186) : error C1059: non constant expression in initialization
08-24 22:08:48.886 30370 30592 E Kodi    :                                                    0(188) : error C1059: non constant expression in initialization
08-24 22:08:48.886 30370 30592 E Kodi    :                                                    0(189) : error C1059: non constant expression in initialization
08-24 22:08:48.886 30370 30592 E Kodi    :                                                    0(191) : error C1059: non constant expression in initialization
...
08-24 22:08:48.886 30370 30592 E Kodi    :                                                    
08-24 22:08:48.904 30370 30592 E Kodi    : 2026-08-24 22:08:48.904 T:30592   error <general>: CShaderGLES::Create: Fragment shader compile error:
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    0(177) : error C7573: OpenGL/ES requires precision specifier on this float type (there is no default precision)
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    0(178) : error C7573: OpenGL/ES requires precision specifier on this float type (there is no default precision)
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    0(180) : error C7573: OpenGL/ES requires precision specifier on this float type (there is no default precision)
...
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    0(497) : error C7553: OpenGL array assignments require #version 120
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    0(501) : error C7553: OpenGL array assignments require #version 120
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    0(644) : error C7532: global type mat2x3 requires "#version 120" or later
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    0(644) : error C0000: ... or #extension GL_NV_non_square_matrices : enable
08-24 22:08:48.904 30370 30592 E Kodi    :                                                    
08-24 22:08:48.906 30370 30592 E Kodi    : 2026-08-24 22:08:48.906 T:30592   error <general>: CShaderGLES::Create: Shader program link error:
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    Vertex info
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    -----------
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(185) : error C1059: non constant expression in initialization
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(186) : error C1059: non constant expression in initialization
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(188) : error C1059: non constant expression in initialization
...
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(217) : error C1059: non constant expression in initialization
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    (0) : error C2003: incompatible options for link
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    Fragment info
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    -------------
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(177) : error C7573: OpenGL/ES requires precision specifier on this float type (there is no default precision)
...
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(185) : error C7573: OpenGL/ES requires precision specifier on this float type (there is no default precision)
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(185) : error C1059: non constant expression in initialization
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(186) : error C7573: OpenGL/ES requires precision specifier on this float type (there is no default precision)
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(186) : error C1059: non constant expression in initialization
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(188) : error C7573: OpenGL/ES requires precision specifier on this float type (there is no default precision)
...
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(331) : error C7553: OpenGL array assignments require #version 120
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(342) : error C7553: OpenGL array assignments require #version 120
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(354) : error C7553: OpenGL array assignments require #version 120
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(365) : error C7553: OpenGL array assignments require #version 120
...
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(644) : error C7532: global type mat2x3 requires "#version 120" or later
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    0(644) : error C0000: ... or #extension GL_NV_non_square_matrices : enable
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    (0) : error C2003: incompatible options for link
08-24 22:08:48.906 30370 30592 E Kodi    :                                                    
08-24 22:08:48.906 30370 30592 E Kodi    : 2026-08-24 22:08:48.906 T:30592   error <general>: CShaderGLES::Create: Failed to load video shader: /data/user/0/org.xbmc.kodi/files/apk/assets/addons/game.shader.presets/resources/glsl/bezel/uborder/base_presets/uborder-bezel-reflections/../../shaders/content_bezel_shaders/crt-nobody-bezel-reflections.glsl
08-24 22:08:48.906 30370 30592 E Kodi    : 2026-08-24 22:08:48.906 T:30592   error <general>: CShaderPresetGLES::CreateShaders: Couldn't create a video shader
08-24 22:08:48.907 30370 30592 W Kodi    : 2026-08-24 22:08:48.907 T:30592 warning <general>: CShaderPreset::SetShaderPreset: Failed to initialize shaders
```

--- a/game.shader.presets/resources/ShaderPresetsGLSLP_GLES.xml
+++ b/game.shader.presets/resources/ShaderPresetsGLSLP_GLES.xml
@@ -116,11 +116,6 @@
         <name>PlayStation Portable 2x</name>
     </preset>
     <preset>
-        <path>glsl/bezel/uborder/base_presets/uborder-bezel-reflections/uborder-bezel-reflections-crt-nobody.glslp</path>
-        <folder>Bezel</folder>
-        <name>Uborder</name>
-    </preset>
-    <preset>
         <path>glsl/borders/bigblur.glslp</path>
         <folder>Borders</folder>
         <name>Big blur</name>

